Marlin/src/inc/Conditionals-4-adv.h

```diff
@@ -745,6 +728,24 @@
   #endif
 #endif
 
```

Comment:
it seems the position of this code is important, so maybe add a comment why it was moved where?

Reply:
"Dear reader, before using `#if` to look at a thing, it helps to `#define` that thing beforehand."